src/extension/server/runmeServer.ts

```diff
@@ -90,10 +87,14 @@ class RunmeServer implements Disposable {
             throw new RunmeServerError('Cannot find server binary file')
         }
 
+        while (!(await isPortAvailable(this.#port))) {
+          this.#port++
```

Comment:
@christian-bromann It is a while loop so it will keep incrementing until it finds a port not in use.

I wouldn't have any problems with using a random port, but we already decided to go with an incremental approach.

Reply:
gotcha